@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgAmmo.hpp

```diff
@@ -248,45 +292,68 @@ class CfgAmmo
 	};
 	class fza_agm114a : fza_agm114k
 	{		
-		fza_salType = "sal1";
-		model = "\fza_ah64_US\fza_agm114a";
-		proxyShape="\fza_ah64_US\fza_agm114a";
-		effectsmissile = "missile3";	
-		hit = 800;
-		indirectHit = 400;
-		indirectHitRange = 10;
-		cost = 1000;
+		fza_salType 				= "sal1";
+		model 						= "\fza_ah64_US\fza_agm114c";
+		proxyShape					= "\fza_ah64_US\fza_agm114c";
+		effectsmissile 				= "missile3";
+
+		submunitionAmmo				= "ammo_Penetrator_Titan_AT";
+
+		hit							= 200;
+		indirectHit 				= 80;
+		indirectHitRange 			= 4;
+
+		//AI usage
+		aiAmmoUsageFlags 			= "128+512";
+		allowAgainstInfantry 		= 0;
+		cost 						= 500;
 	};
 	class fza_agm114c : fza_agm114k
 	{
-		fza_salType = "sal1";
-		model = "\fza_ah64_US\fza_agm114c";
-		proxyShape="\fza_ah64_US\fza_agm114c";		
-		hit= 800;
-		indirectHit = 400;
-		indirectHitRange = 10;
-		cost = 1000;
+		fza_salType 				= "sal1";
+		model 						= "\fza_ah64_US\fza_agm114c";
+		proxyShape					= "\fza_ah64_US\fza_agm114c";
+
+		submunitionAmmo				= "ammo_Penetrator_Titan_AT_long";
+
+		hit							= 260;
+		indirectHit 				= 100;
+		indirectHitRange 			= 4;
+
+		//AI usage
+		aiAmmoUsageFlags 			= "128+512";
+		allowAgainstInfantry 		= 0;
+		cost 						= 600;
 	};
 	class fza_agm114m : fza_agm114k
 	{
-		fza_salType = "sal2";
-		model = "\fza_ah64_US\fza_agm114m";
-		proxyShape ="\fza_ah64_US\fza_agm114m";		
-		hit = 900;
-		indirectHit = 500;
-		indirectHitRange = 20;
-		cost = 1000;
+		ace_frag_enabled 			= 1;
```

Comment:
Your AGM-114M has a penetrator because it inherits it from `fza_agm114k` and therefore `fza_agm114base`. The M is a blast fragmentation warhead and as such has no penetrator.